plugins/woocommerce/tests/legacy/unit-tests/woocommerce-admin/wc-admin-helper.php

```diff
@@ -135,4 +135,23 @@ public function range_provider() {
 			'2 month old store not within 6+ months?'  => array( 2 * MONTH_IN_SECONDS, 'month-6+', false ),
 		);
 	}
+
+	public function test_is_fresh_site() {
```

Comment:
Nice, thanks for adding the test!

How about also adding a test with `fresh_site=0`?

Reply:
Sounds good! I'll add it soon.

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/45232/commits/f7d9c3078926da621806f9e7c019e9919134199f